src/assets/scss/components/comments.scss

```diff
@@ -3,44 +3,86 @@
 .comments-wrapper {
   border-top: 1px solid $eos-gray-100;
   margin-top: 32px;
+
+  .comments-heading {
+    margin-block-end: 0px;
+    margin-block-start: 0px;
+  }
+
 }
 
 .comment {
   display: flex;
   flex-direction: row;
   margin: 8px 0px;
+
+  .avatar {
+    height: fit-content;
```

Comment:
This is giving quite a weird look to the profile pictures. I guess setting height as 2.5rem too will fix.
<img width="948" alt="Screenshot 2022-07-17 at 9 15 05 PM" src="https://user-images.githubusercontent.com/60281688/179406036-04d8d421-9543-4c17-a62e-03a4f1ed489a.png">

Reply:
Done